src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,73 @@
+import Image from "next/image";
+
+import FacebookIcon from "@/assets/image/icons/socialicon/icon_facebook.svg";
+import InstagramIcon from "@/assets/image/icons/socialicon/icon_instagram.svg";
+import XIcon from "@/assets/image/icons/socialicon/icon_X.svg";
+import YoutubeIcon from "@/assets/image/icons/socialicon/icon_youtube.svg";
+
+const socialIcons = [
+  { href: "https://www.facebook.com/", src: FacebookIcon, alt: "Facebook" },
+  { href: "https://www.instagram.com/", src: InstagramIcon, alt: "Instagram" },
+  { href: "https://www.youtube.com/", src: YoutubeIcon, alt: "YouTube" },
+  { href: "https://x.com/", src: XIcon, alt: "X" },
+];
+
+export default function Footer() {
+  return (
+    <footer className="w-full my-5 bg-white text-[13px] text-[color:var(--gray-400)]">
```

Comment:
`text-[color:var(--gray-400)]` 클래스는 `text-gray-400`으로 변경할 수 있어요!

```suggestion
    <footer className="w-full my-5 bg-white text-[13px] text-gray-400">
```